config/default.json

```diff
@@ -2,6 +2,7 @@
   "HOSTNAME": "http://quoting-service",
   "LISTEN_ADDRESS": "0.0.0.0",
   "PORT": 3002,
+  "MONITORING_PORT": 3003,
```

Comment:
Why there is different port for monitoring. I think this is for the health and metrics endpoint in the handler service . May be we can use the same port. I don't think there is a need for introducing a new port here.

Reply:
May be we can follow the same approach of ml-api-handler and notification-handler. Not sure how it is done there.

Reply:
A few reasons:

- ml-api uses a single API server to serve both the ml-api endpoints and the monitoring endpoints. This is not the case for quoting. Quoting uses two separate hapi servers, one for quotes endpoints (with its own health and metrics endpoints) and the other for serving monitoring endpoints for the handlers.

- The implementation in ml-api makes it possible to run the endpoints, monitoring and the handlers at the same time on the same port on a single machine without the need for docker but with quoting's implementation, that is not possible without another port for the monitoring server.

- The monitoring port was introduced to allow for running all the services on a single machine without needing docker. This is helpful for debugging issues that cut across both services without the complexity of docker debugging setup.

- To refactor to align with ml-api's implementation, I reckon, is non-trivial and thus out of scope for #3689.